templates/crate/details.html

```diff
@@ -40,11 +40,17 @@
                         {%- if let Some(source_size) = source_size -%}
                             <li class="pure-menu-heading">Size</li>
                             <li class="pure-menu-item">
-                                <span class="documented-info">Source code size: {{(*source_size)|filesizeformat}}</span>
+                                <span class="documented-info">Source code size: {{(*source_size)|filesizeformat}} <span class="size" tabindex=0>
+                                    {{- crate::icons::IconCircleInfo.render_solid(false, false, "") -}}
+                                    <span class="info">This is the summed size of all downloaded crates.io package files</span>
```

Comment:
I think this is not clear enough. 

It is: summed size of all the files _inside_ the crates.io package for this release.